x-pack/test/usage_collection/test_suites/stack_management_usage/index.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import _ from 'lodash';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { stackManagementSchema } from '../../../../../src/plugins/kibana_usage_collection/server/collectors/management/schema';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  describe('Stack Management', function () {
+    this.tags('ciGroup1');
+    const { common } = getPageObjects(['common']);
+    const browser = getService('browser');
+
+    let registeredSettings: Record<string, any>;
+
+    before(async () => {
+      await common.navigateToApp('home'); // Navigate to Home to make sure all the appIds are loaded
+      registeredSettings = await browser.execute(() => window.__registeredUiSettings__);
+    });
+
+    it('registers all UI Settings in the UsageStats interface', () => {
+      const unreportedUISettings = Object.keys(registeredSettings)
+        .filter((key) => key !== 'buildNum')
+        .filter((key) => typeof _.get(stackManagementSchema, key) === 'undefined');
+
+      expect(unreportedUISettings).to.eql([]);
+    });
```

Comment:
We should add either a comment or an explicit error message (or both) describing how to 'fix' this test when it fails, as it will likely happen when developers add new uiSettings without updating the schema+interface.